classes/spec_extensions.rb

```diff
@@ -85,6 +85,8 @@ def or_license_name_and_url
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
         elsif license.scan(/apache2/).count > 0
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
+        elsif license.scan(/apache 2/).count > 0
```

Comment:
@segiddins then are we okay no longer matching the string "Apache2" as valid? The regex you posted won't match that. At the very least, it should be changed to `/apache (license, version )?2(\.0)?/i` (adding the ? after the first group), so that it can match the shorter forms like "Apache 2" or "Apache 2.0" and doesn't require that the sequence "apache " be followed by "license, version 2". Here's how your regex will match: http://www.regexpal.com/?fam=94156